torch/_tensor_docs.py

```diff
@@ -1453,9 +1453,24 @@ def callable(a, b) -> number
 
 add_docstr_all('stride',
                """
-stride() -> tuple
+stride(dim) -> tuple or int
 
 Returns the stride of the tensor.
+Stride is the jump necessary to go from one element to the next one in the specified dimension dim.
+Tuple is returned when no Argument is passed. So we get stride in all dimensions.
+Integer value is returned when we desire stride in particular dimension.
+
+Args:
+    dim (int): The desired dimension in which stride is required.
+
+Example:
+>>> x = torch.Tensor([[1, 2, 3, 4, 5], [6, 7, 8, 9, 10]])
```

Comment:
you need to replace the tabs with 4 spaces. right now you removed the spaces entirely.

Reply:
:smile: this time i cleared all the tests. but i have pushed the above said changes.

Reply:
This seems to be a bug. Should I open an issue regarding this.